cli/Sources/Noora/Localization.swift

```diff
@@ -0,0 +1,148 @@
+import Foundation
+
+/// A struct that encapsulates localized alert titles and recommended titles for different types of alerts.
+public struct Localization {
```

Comment:
What about this? I think it's more accurate to name it `Locales`:

```suggestion
public struct Locales {
```